addons/sale/report/sale_report.py

```diff
@@ -236,3 +240,13 @@ def _query(self):
     @property
     def _table_query(self):
         return self._query()
+
+    def action_open_record(self):
```

Comment:
```suggestion
    def action_open_order(self):
```
since you are opening sales order not report record